CoffeeChat/Controller/HomeViewController.swift

```diff
@@ -91,6 +93,31 @@ class HomeViewController: UIViewController {
         }
     }
 
+    private func setUpLocalNotifications() {
+        let center = UNUserNotificationCenter.current()
+        center.requestAuthorization(options: [.alert, .sound]) { (granted, error) in
+            print("granted: \(granted)")
+        }
+        let content = UNMutableNotificationContent()
+        content.title = "Meet your new pear!"
+        content.body = "Set up this week's chat today 😊"
+        let date = setNotificationDate()
+        let dateComponents = Calendar.current.dateComponents([.year, .month, .day, .hour, .minute, .second], from: date)
+        let notification = UNCalendarNotificationTrigger(dateMatching: dateComponents, repeats: false)
+        let uuid = UUID().uuidString
+        let request = UNNotificationRequest(identifier: uuid, content: content, trigger: notification)
+        center.add(request)
+    }
+
+    private func setNotificationDate() -> Date {
+        let formatter = DateFormatter()
+        formatter.dateFormat = "yyyy/MM/dd HH:mm"
+        // set the notification date and time
+        let dateString = "2021/02/15 08:00"
```

Comment:
Why is this date hard coded? Also, unless you're decoding some externally obtained date (user input, server response, etc.), I would directly construct the date with a `Calendar` and `DateComponent`s rather than going through a date formatter.